src/xenia/debug/ui/debug_window.cc

```diff
@@ -959,6 +959,8 @@ void DebugWindow::DrawRegistersPane() {
     case RegisterGroup::kHostVector: {
       ImGui::BeginChild("##host_vector");
       for (int i = 0; i < 16; ++i) {
+        float f[4];
```

Comment:
ah, interesting. We aren't using xmm_registers as xmm registers, so they should be `vec128_t` instead. Then we wouldn't need to do this dance to get the values.